docs/codeql/codeql-overview/system-requirements.rst

```diff
@@ -26,9 +26,9 @@ For extraction of compiled languages (C/C++, C#, Go, Java) and Ruby on Linux:
 - ``glibc`` version 2.17 or greater must be installed.
 - ``musl-c``-based Linux distributions, such as Alpine Linux, are not supported.
 
-For TypeScript extraction on all platforms:
+For JavaScript and TypeScript extraction on all platforms:
```

Comment:
```suggestion
For TypeScript extraction on all platforms:
```

Node.js is only used when we extract TypeScript. It's not a requirement for extraction of a pure JS repository.